src/pages/partners/deriv-prime/index.tsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import { WithIntl } from 'components/localization/withIntl'
+import Layout from 'components/layout/layout'
```

Comment:
please don't use this layout component, use the one we have in features folder, thanks :pray:

Reply:
okay got it